src/Diagnostics/VarianceDissipationComputations/assemble_dissipation.jl

```diff
@@ -0,0 +1,67 @@
+function assemble_dissipation!(model, dissipation)
+
+    for tracer_name in keys(dissipation.advective_production)
+        assemble_dissipation!(dissipation, model, tracer_name)
+    end
+
+    return nothing
+end
+
+@inline c★(i, j, k, grid, cⁿ⁺¹, cⁿ) = @inbounds (cⁿ⁺¹[i, j, k] + cⁿ[i, j, k]) / 2
+@inline c²(i, j, k, grid, cⁿ⁺¹, cⁿ) = @inbounds (cⁿ⁺¹[i, j, k] * cⁿ[i, j, k])
+
+# Assemble the dissipation from the previously calculated advective and 
+# diffusive fluxes, the formulation is:
+#
+# A = 2 * δc★ * F - U δc²  # For advective dissipation
+# D = 2 * δc★ * F           # For diffusive dissipation
+#
+# Where ``F'' is the flux associated with the particular process,``U'' is the adecting velocity,
+#  while ``c★'' and ``c²'' are functions defined above.
+# Note that ``F'' and ``U'' need to be numerically accurate for the budgets to close,
+# i.e. for and AB2 scheme:
+#
+# F = 1.5 Fⁿ - 0.5 Fⁿ⁻¹
+# U = 1.5 Uⁿ - 0.5 Uⁿ⁻¹
+#
+# For an RK3 method (not implemented at the moment), the whole substepping procedure needs to be accounted for.
```

Comment:
Can we make this into a docstring?

Reply:
Also, it's not clear to me what "assemble" means. Could you please clarify?

Reply:
I have changed the name of the function to compute